components/style/properties/properties.mako.rs

```diff
@@ -1328,10 +1329,26 @@ impl ToCss for PropertyDeclaration {
 }
 
 impl MallocSizeOf for PropertyDeclaration {
-    fn malloc_size_of_children(&self, _malloc_size_of: MallocSizeOfFn) -> usize {
-        // The variants of PropertyDeclaration mostly (entirely?) contain
-        // scalars, so this is reasonable.
-        0
+    fn malloc_size_of_children(&self, malloc_size_of: MallocSizeOfFn) -> usize {
+        match *self {
+            % for property in data.longhands:
+                % if property.boxed and property.is_vector:
+                    (this should not happen! it is not smart to box a vector here)
```

Comment:
I was going to suggest `compile_error!("this should not ...");` but that macro is only available in Rust 1.20.

Actually, probably better to do `% raise Exception("this should not")` so that the mako step fails rather than the Rust compilation step.